app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -44,12 +44,6 @@ class Vmware::InfraManager < InfraManager
       unsupported_reason_add(:streaming_refresh, "Streaming refresh not enabled") unless streaming_refresh_enabled?
     end
 
-    has_many :host_guest_devices, :through => :host_hardwares, :source => :guest_devices
-    has_many :miq_scsi_targets, -> { distinct }, :through => :host_guest_devices
-    has_many :miq_scsi_luns, -> { distinct }, :through => :miq_scsi_targets
-    has_many :host_system_services, :through => :hosts, :source => :system_services
-    has_many :distributed_virtual_switches, :dependent => :destroy, :foreign_key => :ems_id
```

Comment:
This also existed in core already https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/infra_manager.rb#L17